src/improvements/README.md

```diff
@@ -1,59 +1,200 @@
+Tasks are organized by network to maintain clear separation and specific requirements:
 
-Developers can now create tasks without writing any Solidity code as long as a predefined template is used. The templates are designed to be as flexible as possible, allowing developers to create tasks that update multiple parameters for a single or super chain. Templates are currently in development and will be released in a future PR.
+1. **Ethereum Mainnet** (`tasks/eth/`)
+   - Production-ready tasks
+   - Example: eth/001-security-council-phase-0
 
-# Task Configuration File
+2. **Sepolia Testnet** (`tasks/sep/`)
+   - Testing and validation
+   - Development environment
+   - Example: sep/001-op-extended-pause
 
-The template configuration file is `config.toml` which contains the network configurations to modify on L1.
+3. **Optimism Networks**
+   - `tasks/oeth/`: Optimism Ethereum tasks
+   - `tasks/opsep/`: Optimism Sepolia tasks
+   - Network-specific configurations
 
-### L2Chains
+## Quick Start
 
+1. Create a new task:
+```bash
+cd src/improvements/
+just new task
+```
+
+2. Configure the task in `config.toml`:
 ```toml
-# L2Chains is a list of the L2 chains that the task will interact with
-l2chains = [{"name": "Base", "chainId": 8453}, {"name": "Metal", "chainId": 1750}, {"name": "OP Mainnet", "chainId": 10}]
+templateName = "GasConfigTemplate"
+l2chains = [{"name": "OP Mainnet", "chainId": 10}]
 ```
 
-### Task Template Files
+3. Test the task:
+```bash
+forge script <template-path> --sig "run(string)" <config-path> --rpc-url devnet -vvv
+```
 
-Task templates allow task developers to create new tasks without writing any Solidity code. The templates are designed to be as flexible as possible, allowing developers to create tasks that update multiple parameters for a single or superchain. Templates are currently in development and will be released in a future PR.
+## Available Templates
 
-The mainnet configuration file for task template example 00 can be found here: [task-00/mainnetConfig.toml](./example/task-00/mainnetConfig.toml)
+1. **Gas Config Template**
+   - Set gas limits for L2 chains
+   - Configure transaction parameters
+   - Example: [Gas Config Example](#gas-config-example)
 
-```toml
-[gasConfigs]
-gasLimits = [{chainId = 291, gasLimit = 100000000}, {chainId = 1750, gasLimit = 100000000}]
-```
+2. **Dispute Game Template**
+   - Configure dispute game implementations
+   - Set game parameters
+   - Example: [Dispute Game Example](#dispute-game-example)
 
-This toml configuration file allows task developers to set gas limits for the task. After the changes are applied in the simulation, validations are run to ensure that the new values are the expected values.
+3. **Game Type Template**
+   - Set respected game types
+   - Configure chain permissions
+   - Example: [Game Type Example](#game-type-example)
 
-### Running Example Tasks
+## Documentation
 
-#### Template 00 to set gas configs:
+### Core Concepts
```

Comment:
I'd delete this section, you're linking to these files at the top of the file already.